eng/scripts/Invoke-DependencyCheck.ps1

```diff
@@ -0,0 +1,77 @@
+Param(
+    [string] $newPackageDirectory
+)
+
+. (Join-Path $PSScriptRoot .. common scripts common.ps1)
+
+$ignoreCheck = "sdk/synapse/azartifacts", "sdk/template/aztemplate", "sdk/azcore", "sdk/azidentity"
+
+$sdks = Get-AllPackageInfoFromRepo
+
+$sdkRoot = Join-Path $RepoRoot "sdk"
+
+# Get all existed packages
+$packagesImport = ""
+$newPackage = ""
+foreach ($sdk in $sdks)
+{
+    if ($sdk.Name -like "*internal*" -or $sdk.Name -in $ignoreCheck)
+    {
+        continue
+    }
+    if ((Resolve-Path $sdk.DirectoryPath).Path -eq (Resolve-Path (Join-Path $sdkRoot $newPackageDirectory)).Path)
+    {
+        $newPackage = $sdk.Name
+    }
+    $packagesImport = $packagesImport + "`t_ `"github.com/Azure/azure-sdk-for-go/$($sdk.Name)`"`n"
```

Comment:
Do we need to specify versions of these dependencies, or are we fine with taking the latest version?

Reply:
For single package, the script will let `go mod tidy` to try to find all compatible version except using `replace` to use the latest code of THE package which is under ci check.